packages/relative-values/src/components/ui/icons/Icon.tsx

```diff
@@ -4,18 +4,20 @@ export type IconProps = PropsWithChildren<{
   size?: number;
   className?: string;
   onClick?: () => void;
+  viewBox?: string;
```

Comment:
This prop shouldn't be exposed: it's a property of an icon component (e.g. `ScaleIcon` or `CompassIcon`), not of the way an icon is used.

Possible fix: remove this prop from `IconProps`, but add it to `Icon` props below (`Icon: FC<IconProps & { viewBox?: string }>`, and then pass it in `ScaleIcon` and `CompassIcon` definitions).

Another, better solution, is to rescale SVG before exporting from Figma, so that `0 0 20 20` is ok for all definitions.